### PR TITLE
i3 add rspec

### PR DESCRIPTION
**Pull request by duleorlovic**
- [#3] rails g rspec:install
- [#3] standardrb --fix spec/
- [#3] git mv test/fixtures/ spec/
- [#3] git rm -rf test/

### CODE REVIEW

**Comment by duleorlovic** [comment]
@trkin/internship I think it is good to have separate commit when we just run the commands (we need to put that command in commit message)

I put this guide in https://dev.trk.in.rs/2023/03/21/git-tips/#git-style

I think it is safe to merge this since I did not change any file manually, I just run the commands you can find in commit messages

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Checks for pending migrations and applies them before tests are run.
+# If you are not using ActiveRecord, you can remove these lines.
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
+  abort e.to_s.strip
+end
+RSpec.configure do |config|
+  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
+  config.fixture_path = "#{::Rails.root}/spec/fixtures"
```
Comment (DeVuDeveloper) [inline spec/rails_helper.rb:34]: Nice job @duleorlovic.
Do we have now fixtures? Can I seed for example users with them?

Comment (duleorlovic) [reply spec/rails_helper.rb:34]: Yes, I'm waiting for this to be approved and merged and I will create fixtures.

Comment (DeVuDeveloper) [reply spec/rails_helper.rb:34]: Sorry, I am not sure if the fixture works. It was a nice job so far, but try to make one fast test using fixtures.
I need to be sure that the configuration is ok without warnings.
Also, try to seed Users with fixtures.
Happy coding.

Comment (duleorlovic) [reply spec/rails_helper.rb:34]: @DeVuDeveloper I need this merged in order to submit seed example, can you approve ?

Comment (DeVuDeveloper) [reply spec/rails_helper.rb:34]: Hi, @duleorlovicj👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!!   There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉

**Review by DeVuDeveloper — APPROVED** [review]

**Pull request merged**